### ISSUE TITLE
Tool C. 11. – Validate CWD Layer

### ISSUE THREAD

**Issue by gahisu** [USDA-NRCS/Wetland-Tools---ArcGIS-Pro#13]
When validation fails due to missing attributes, the Site_CWD layer cannot be edited due to a file lock. A work-around is to remove the layer from the Contents and then re-add using the Undo last action button; the layer can then be edited.

**Comment by CEM-NRCS** [comment]
Another workaround is to close the map frame and reopen it from the catalog pane without removing/undoing removal of layers. Does this alternate workaround method work for you instead/as well?

**Comment by lmblackburn** [comment]
I also got the missing paramaters message when trying to use the Site_CWD layer, no matter how many times I closed and opened the map pane.
It seems like the directions in the word document were wrong.  It siad to select the Site_CWD layer...but the tool directs the user to select the Site sampling Units layer....when I used this layer I didn't get the missing paramaters message.  Though I didn't get the message "all checks passed" in the tool details.  Here's what my messages looked like:
 
![image](https://user-images.githubusercontent.com/87446239/125954184-5a17fe7d-6628-4fd7-95a6-f4b0a614de2e.png)
